frontend/src/components/samples/SampleDetailProcess.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+
+import {Table, Tag} from "antd";
+
+import {withUser, withProtocol} from "../../utils/withItem";
+
+
+const mapStateToProps = state => ({
+  usersByID: state.users.itemsByID,
+  protocolsByID: state.protocols.itemsByID,
+});
+
+const SampleDetailProcess = ({processes, usersByID, protocolsByID,}) => {
+    const columns = [
+      {
+        title: '',
+        dataIndex: 'id',
+        key: 'id',
+        render: (id, _) =>
+          <Link to={`/processes/${id}`}>
+            Process #{id}
```

Comment:
The process ID provided to the user is the process model id, the process_sample model id is kept internal (you use it to link to the right processes, but do not display that id) and it is not provided to the user since it has no use to the user and would just confuse him (what is the difference with process and process_sample what is that id is it process or process_sample ?).

Reply:
We are back to the argument of how do we identify a 'process-sample' . I thought on your end you decided to name it Process #23424  as a mean to represent ProcessSample #23424. As I have said, this might get confusing very quickly, so we better figure out a way to represent those concepts in our interface, and not confuse it with our model names :)

Reply:
This is more of a stopgap solution. The idea is to eventually have this process ID lead to a process"batch" ID page where you can see the meta-data at that level. There is no real meta-data there at the moment and since we have not found/decided how to link the sample level data this is as good a solution as any without introducing a new "entity" which is not really one ;) (processSample). The eye / "details..." / repr of processSample still has my vote.